spec/services/patient_breakdown_service_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe PatientBreakdownService do
+  let!(:facility) { create(:facility) }
+  let!(:ltfu_patient) { create(:patient, :hypertension, recorded_at: 2.years.ago, assigned_facility: facility) }
+  let!(:not_ltfu_patient) { create(:patient, :hypertension, assigned_facility: facility) }
+  let!(:ltfu_transferred_patient) { create(:patient, :hypertension, recorded_at: 2.years.ago, status: :migrated, assigned_facility: facility) }
+  let!(:not_ltfu_transferred_patient) { create(:patient, :hypertension, status: :migrated, assigned_facility: facility) }
+  let!(:dead_patient) { create(:patient, :hypertension, status: :dead, assigned_facility: facility) }
+  let!(:not_hypertensive_patient) { create(:patient, :without_hypertension, assigned_facility: facility) }
+  let!(:blood_pressures) {
+    [create(:blood_pressure, recorded_at: 1.day.ago, patient: not_ltfu_patient),
+      create(:blood_pressure, recorded_at: 1.day.ago, patient: not_ltfu_transferred_patient)]
+  }
```

Comment:
You can create all these models right in the `it` block itself ..there is just one test here after all.  No reason to scatter all this state into `let!`.